<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reflections_style_editor</class>
 <widget class="QWidget" name="reflections_style_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Series Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="offset_layout">
     <item>
      <widget class="QLabel" name="offset_label">
       <property name="text">
        <string>Offset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ScientificDoubleSpinBox" name="offset_x">
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ScientificDoubleSpinBox" name="offset_y">
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="symbol_label">
     <property name="text">
      <string>Symbol:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Outline Thickness:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="symbol">
     <item>
      <property name="text">
       <string>o</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>s</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>t</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>t1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="fill">
     <property name="text">
      <string>Fill</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ScientificDoubleSpinBox" name="outline_thickness">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="minimum">
      <double>0.000010000000000</double>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
     <property name="value">
      <double>3.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ScientificDoubleSpinBox" name="point_size">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="minimum">
      <double>0.000010000000000</double>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.py</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>symbol</tabstop>
  <tabstop>point_size</tabstop>
  <tabstop>outline_thickness</tabstop>
  <tabstop>fill</tabstop>
  <tabstop>offset_x</tabstop>
  <tabstop>offset_y</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
